src/main/java/com/google/ehub/servlets/LoginServlet.java

```diff
@@ -22,28 +24,20 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
+    JsonObject loginJson = new JsonObject();
 
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/?authuser=0";
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
-
-      response.setContentType("text/html");
-      response.getWriter().println("<p>Hello " + userEmail + "!</p>");
-      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+      String logoutUrl = userService.createLogoutURL("/?authuser=0");
+      loginJson.addProperty("LogoutURL", logoutUrl);
+      loginJson.addProperty("LoginURL", "");
```

Comment:
Why do we need to add an empty string for the property "LoginURL"? Is it possible to simply remove that line?

Reply:
I have it as an empty string so that there will be two fields in the json

Reply:
I think Bryan is asking if you need both fields in the json. Can you just add one URL field?